.github/README.md

```diff
@@ -0,0 +1,55 @@
+# [Oppia-Android](https://www.oppia.org)
+
+Oppia-Android is a mobile application version of [Oppia](https://www.oppia.org). It is an android application that enables anyone to easily learn interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
+
+In addition to that users can create their profile, pratice a set of skills, play with exporations, review previously finished skills and can also track the progress.
+
+Oppia-Android app is written using Kotlin and python, and is built on top of Google App Engine.
+
+  * [Oppia.org community site](https://www.oppia.org)
+  * [User Documentation](https://oppia.github.io/)
+  * [Contributors' wiki](https://github.com/oppia/oppia-android/wiki)
+  * [Developer mailing list](http://groups.google.com/group/oppia-android-dev)
+  * [File an issue](https://github.com/oppia/oppia-android/issues/new/)
+
+<p align="center">
+  <img width="361" alt="Screenshot 2019-12-18 at 4 37 57 PM" src="https://user-images.githubusercontent.com/9396084/71081269-ddf6f280-21b4-11ea-9a49-fcbd65b4ced7.png">
+</p>
+
+
+## Installation
+
+Please refer to the [developer wiki](https://github.com/oppia/oppia-android/wiki) for full installation instructions. This is just a short summary for developers who would like to contribute:
+
+1. Create a new, empty folder called `opensource/` within your home folder. Navigate to it (`cd opensource`), then [fork and clone](https://github.com/oppia/oppia-android/wiki/Fork-and-Clone-Oppia) the Oppia repo. This will create a new folder named `opensource/oppia`.
+
+2. Navigate to `opensource/oppia-android/` and run the project in android studio.
+
+## Contributing
+
+The Oppia-Android project is built by the community for the community. We welcome contributions from everyone, especially new contributors.
+
+You can help with Oppia-Android's development in many ways, including art, coding, design and documentation.
+  * **Developers**: please see [this wiki page](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change) for instructions on how to set things up and commit changes.
+  * **All other contributors**: please see our [general contributor guidelines](https://github.com/oppia/oppia-android/wiki).
+
+
+## Support
+
+If you have any feature requests or bug reports, please log them on our [issue tracker](https://github.com/oppia/oppia-android/issues/new).
+
+Please report security issues directly to admin@oppia.org.
+
+
+## License
+
+The Oppia-Android code is released under the [Apache v2 license](https://github.com/oppia/oppia-android/blob/develop/LICENSE).
+
+
+## Keeping in touch
+
+  * [Blog](https://medium.com/oppia-org)
+  * [Discussion forum](http://groups.google.com/group/oppia-android)
```

Comment:
Does this link need access permission.

Reply:
I believe yes. But I think these links needs to be verified by @BenHenning and @seanlip once I pass it to them

Reply:
Okay, I tried to open this link and it asked for permission or it said the group doesn't exist.

Reply:
Thanks.